pages/bridge.tsx

```diff
@@ -130,11 +131,33 @@ const Bridge = () => {
                 <h3 className="gray-600 font-bold">From</h3>
                 <div>{source.chain.name}</div>
                 <div className="mt-3">
-                  <span className="gray-600 font-bold">
-                    {mounted && source.token ? source.token.symbol : 'Token'} balance
-                  </span>
+                  <span className="gray-600 font-bold">Balance</span>
                   <br />
-                  {mounted && source.token ? source.token.formatted : '0.00'}{' '}
+                  <div className="items-center flex">
+                    {mounted && source.token ? source.token.formatted : '0.00'}{' '}
+                    <div className="ml-5 gray-600 font-bold">
+                      {mounted && source.token ? source.token.symbol : 'Token'}
+                    </div>
+                    <Image
+                      className="ml-2"
+                      src={source.tokenLogo}
+                      width={40}
+                      height={40}
+                      alt={`${source.token?.symbol} logo`}
+                    />
+                    {source.chain.id === 420 && (
+                      <>
+                        <div className="flex justify-end -ml-3 -mb-8 z-10">🗳️</div>
+                        <Image
+                          className="-ml-12 -mt-8"
+                          src={source.chainLogo}
+                          width={16}
+                          height={16}
+                          alt={`${source.chain.name} logo`}
+                        />
+                      </>
+                    )}
```

Comment:
This would probably be good to abstract into a separate component because it is used twice and it will be user to modify when we support more chains.

Reply:
@alexkeating i can agree; perhaps we make a separate issue for that so as not to block merge here

Reply:
Sounds good, add a link to that issue here and we can mark this as resolved.

Reply:
Created an issue: https://github.com/ScopeLift/l2-flexible-voting-frontend/issues/111